### PR TITLE
fix(design-tokens): DLT-2135 DLT-139 nix references for shadow design tokens 

### PR DESCRIPTION
**Pull request by francisrupert**
# Nix references for shadow design tokens

## :hammer_and_wrench: Type Of Change

- [x] Fix

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-2135
https://dialpad.atlassian.net/browse/DLT-139

## :book: Description

[DLT-2135](https://dialpad.atlassian.net/browse/DLT-2135)
* Because shadow token colors are wrapped in `rgba()`, and that is a css specific function, they are breaking when being transferred to non CSS based platforms (android, iOS). 

[DLT-139](https://dialpad.atlassian.net/browse/DLT-139)
* Bumped the border color, referencing actual border color tokens, and walked back the border width. Did so for input, select, checkbox, radio. The border size was `150` (1.5px) which we're less a fan of by now; besides, it only works on higher pixel-density screens which there's little chance that customers will have anyway.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

For all CSS changes:

- [x] I have used design tokens whenever possible.
- [x] I have considered how this change will behave on different screen sizes.
- [x] I have visually validated my change in light and dark mode.
- [ ] I have used gap or flexbox properties for layout instead of margin whenever possible.

## :crystal_ball: Next Steps

@braddialpad presumably to consider Style Dictionary transforms and resolve iOS/Android issues.

[DLT-2135]: https://dialpad.atlassian.net/browse/DLT-2135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DLT-139]: https://dialpad.atlassian.net/browse/DLT-139?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/dialtone-tokens/tokens/$metadata.json [hunk 1]
```diff
@@ -16,9 +16,7 @@
     "components/badge/dark",
     "components/button/default",
     "components/checkbox/default",
-    "components/checkbox/dark",
```
Comment (francisrupert) [inline packages/dialtone-tokens/tokens/$metadata.json:19]: This and others like it below is intentional. With the border color update made to default, its same values apply to dark too.

**Review by juliodialpad — APPROVED** [review]
Nothing seems wrong code-wise and visual tests passed correctly, thanks!

**Comment by braddialpad** [comment]
Going to check this is working as expected on my branch.

**Review by braddialpad — APPROVED** [review]
Yep looks good now thanks! There are a couple remaining issues with iOS syntax but I will fix them in a separate PR.

**Pull request merged**